common/components/TaskCard/index.tsx

```diff
@@ -36,7 +36,8 @@ const TaskCard = ({ taskId, task }) => {
 
   const taskDelete = (id) => {
     deleteTask(id)
-    router.push(AppRoutes.TASK)
+    // router.push(AppRoutes.TASK)
+    router.push(`/task/user/${user?._id}`)
```

Comment:
change it to ```router.push(`${AppRoutes.TASK}/user/${user?._id}`)``` or create a new `enum` with different URLs and use it, or extend the existed `AppRoutes` `enum`